Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ubuntu:16.04
+
+# Install commands used in our scripts
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update -qq && apt-get install -qq --assume-yes --no-install-recommends sudo apt-utils git wget curl lsb-release
+
+# Add amigo user
+RUN adduser --disabled-password --gecos "" amigo
+RUN adduser amigo sudo
+RUN echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
+USER amigo
+WORKDIR /home/amigo
+
+# Remove interactive check from bashrc
+RUN sed -e s/return//g -i ~/.bashrc
+
+# Setup tue env
+RUN mkdir -p ~/.tue
+ADD ./ /home/amigo/.tue/
```

Comment:
What does this do? AFAIK ```ADD``` adds something from the computer running the Dockerfile, right? Is this then reproducible?